modules/learn/pages/clusters-and-availability/xdcr-enable-crossclusterversioning.adoc

```diff
@@ -0,0 +1,46 @@
+= XDCR enableCrossClusterVersioning
+:description: pass:q[Hybrid Logical Vector (HLV) must be enabled for each of the buckets participating in the replication process using the `enableCrossClusterVersioning` property. As a result, mobile and XDCR become a part of the same replication process.]
+
+[abstract]
+{description}
+
+
+When you set the bucket property `enableCrossClusterVersioning` (ECCV) to `true`, for each document processed by XDCR, XDCR stores an internal history of the document in the extended attributes. This set of information is also called Hybrid Logical Vector (HLV), which is a set of Hybrid Logical Clock (HLC) information.
+
+NOTE: The bucket property `enableCrossClusterVersioning` must be set to `true` to use the `mobile=[Off | Active]` flag, and this flag in turn  enables the setting _XDCR Active-Active with Sync Gateway 4.0+_. For more information about XDCR Active-Active with Sync Gateway 4.0+, see xref:learn:clusters-and-availability/xdcr-active-active-sgw.adoc[XDCR Active-Active with Sync Gateway].
+
+[#hlv-data-maintained-in-xattr]
+== Hybrid Logical Vector (HLV) data maintained in the extended attributes
+
+The new metadata, HLV, is stored as a system user created extended attribute (xattr). The HLV metadata is also called Version Vectors. The system xattr name is `"_vv" -- "xattrs"."_vv"`. 
+
+The HLV metadata takes up `109 + 40N` bytes of space per document, where N is the number of buckets mutating the document in the replication topology. 
+
+NOTE: The size of the HLV metadata must remain constant as long as your replication topology is constant. However, the HLV data accumulation occurs when the replication topology changes and the document copy goes through different clusters, for example, when being restored to a new cluster.
```

Comment:
The size of the HLV metadata can be less than `109 + 40N` -- it'll grow to that limit as long as the replication topology does not change.   So, change to:

NOTE:  As long as your replication topology is constant, the size of the HLV metadata will grow to `109 + 40N` and remain constant.  However, the HLV data accumulation occurs when the replication topology changes and the document copy goes through different clusters, for example, when being restored to a new cluster.